modules/gdnative/include/gdnative/basis.h

```diff
@@ -46,88 +46,17 @@ typedef struct {
 } godot_basis;
 #endif
 
-// reduce extern "C" nesting for VS2013
```

Comment:
We don't support VS 2013 anymore, so if all compilers we support are fine with it, we could drop the close / reopen around includes. Doesn't hurt though.

Reply:
I left it mostly because it was already there. I guess we can drop it now.